<launch>
    <rosparam command="load" file="$(find classic_robot)/param/topics.yaml" />
    <rosparam command="load" file="$(find classic_robot)/param/misc.yaml" />

    <rosparam command="load" file="$(find classic_robot)/param/joy.yaml" />

    <node name="high_level_state_controller" type="high_level_state_controller.py" package="classic_robot"/>
</launch>